pom.xml

```diff
@@ -359,7 +360,12 @@
         <plugin>
           <groupId>org.jacoco</groupId>
           <artifactId>jacoco-maven-plugin</artifactId>
-          <version>0.8.12</version>
+          <version>0.8.13-SNAPSHOT</version>
```

Comment:
SNAPSHOT dependency?

Reply:
because of https://github.com/jacoco/jacoco/pull/1692 
I'm deploying it into our private nexus so it's available in our Jenkins

Reply:
Ok, but would I be able to make a release if I merge this PR?
Aren't SNAPSHOT dependencies forbidden in releases?

Reply:
nah because it's activated only with the `jacoco` profile.